### PR TITLE
AGP-131: Aragon Association acquisition of ANJ for security purposes

### PR DESCRIPTION
**Pull request by luisivan**

### CODE REVIEW

**Comment by luisivan** [comment]
Discussion at https://forum.aragon.org/t/agp-discussion-aragon-association-acquisition-of-anj-for-security-purposes/1487

File: AGPs/AGP-X.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+## Purpose of the transfer
+
+With the imminent launch of the Aragon Court, there may be 51% attacks to the Court itself, which could put the whole Court in jeopardy.
+
+In order to prevent that, we want to propose the Aragon Association to acquire ANJ and hold it over the course of the first few months that the Aragon Court will be operating, until it’s proven and seamlessly working. It would work this way:
+
+- The Aragon Association would participate on the ANJ sale like any other participant and purchase 1m ANT worth of ANJ
+- The acquired ANJ would be stored in the Aragon multisig, controlled by the Aragon Association Board of Directors
+- The acquired ANJ would be gradually sold for ANT and transferred back into the project’s multisig at a rate of 1/6 per month
+- The acquired ANJ would only be used under force majeaure circumstances that would impede Aragon Court from working correctly
```
Comment (LouisGrx) [inline AGPs/AGP-X.md]: 'force majeure'

**Comment by luisivan** [comment]
I am by no means convinced that this AGP makes sense since key stakeholders haven’t chimed in and answered some of the most important questions, but I’ll mark it as final for now and we can always withdraw further on.

**Comment by luisivan** [comment]
Withdrawing this AGP from ANV-5 on behalf of the Aragon Association

**Comment by john-light** [comment]
With the approval of [AGP-155](https://github.com/aragon/AGPs/blob/master/AGPs/AGP-155.md) in Aragon Network Vote #6, the AGP process has now been sunset. All open AGP pull requests are now being closed.

Thank you for participating in the AGP process!

**Pull request closed without merging**